.github/workflows/advanced-security.yml

```
@@ -0,0 +1,630 @@
+      # Upload IaC security results
+      - name: Upload Checkov scan results to GitHub Security
+        uses: github/codeql-action/upload-sarif@v3
+        if: always()
+        with:
+          sarif_file: checkov-results.sarif
+
+  # Container security (if Dockerfile exists)
+  container-security:
+    name: Container Security
+    runs-on: ubuntu-latest
+    if: hashFiles('**/Dockerfile*') != ''
```

actionlint (1.7.7)

calling function "hashFiles" is not allowed here. "hashFiles" is only available in "jobs.<job_id>.steps.continue-on-error", "jobs.<job_id>.steps.env", "jobs.<job_id>.steps.if", "jobs.<job_id>.steps.name", "jobs.<job_id>.steps.run", "jobs.<job_id>.steps.timeout-minutes", "jobs.<job_id>.steps.with", "jobs.<job_id>.steps.working-directory". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)